packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -116,16 +108,15 @@ export interface IOverflowListProps<T> extends Props {
 }
 
 export interface IOverflowListState<T> {
-    /**
-     * Direction of current overflow operation. An overflow can take several frames to settle.
-     *
-     * @internal don't expose the type
-     */
-    direction: OverflowDirection;
+    /** Whether repartitioning is still active. An overflow can take several frames to settle. */
+    repartitioning: boolean;
     /** Length of last overflow to dedupe `onOverflow` calls during smooth resizing. */
     lastOverflowCount: number;
     overflow: readonly T[];
     visible: readonly T[];
+    /** Pointer for the binary search algorithm used to find the finished non-overflowing state */
+    chopSize: number;
+    lastChopSize: number | null;
```

Comment:
I don't think either of these need to live in state, they can just be private members of the component class. They are not referenced in any lifecycle method, just `repartition()`, so it should be save to move out of state.

Reply:
This is not a huge deal, though, so I can fix it in a follow-up PR if necessary. Thanks for the contribution @matt-gold!